packages/test262-runner/test262/test/built-ins/Compartment/prototype/Symbol.toStringTag.js

```diff
@@ -1,7 +1,7 @@
 /*---
 description:
 includes: [propertyHelper.js]
-features: [Symbol.toStringTag,Compartment]
+features: [Symbol.toStringTag,Compartment,ses-xs-parity]
```

Comment:
This is the one XS test that I’ve so far attempted to get working for both Node.js and XS.